src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -660,7 +661,7 @@ void DoorLockServer::setCredentialCommandHandler(
     case DlDataOperationType::kClear:
     default:
         // appclusters, 5.2.4.40: set credential command supports only Add and Modify operational type.
-        sendSetCredentialResponse(commandObj, commandPath, DlStatus::kInvalidField, 0, nextAvailableCredentialSlot);
+        sendSetCredentialResponse(commandObj, commandPath, DlStatus::kInvalidField, userIndex.Value(), nextAvailableCredentialSlot);
```

Comment:
This doesn't make sense.  We're sending an error response here...

Reply:
Oh. It was not clear to me that is an error response. Just reread the comment. I will put this one back as 0.

Reply:
> It was not clear to me that is an error response.

See the status: `DlStatus::kInvalidField`